source/sphinx_extensions/plugin_directory/templates/plugin.rst

```diff
@@ -27,6 +27,30 @@
            {% for line in plugin.user_support_text.splitlines() %}
            {{ line|urlize }}<br/>
            {% endfor %}
+       <tr>
+         <th scope="row">q2cli Invocation</th>
+         <td>
+
+.. command-block::
+
+             qiime {{ title }}
+
+.. raw:: html
+
+         </td>
+       </tr>
+       <tr>
+         <th scope="row">Artifact API Import</th>
+         <td>
+
+.. command-block::
+
+             from qiime2.plugins import {{ title }}
```

Comment:
This has the same hyphenation problem as the import above. Unfortunately you will probably need to just call `.replace(...)` as popping off two "segments" of the import path requires about as much "internal" knowledge about the way the Artifact API is structured as replacing the hyphens.

Something that would make everyone's lives easier is if there was a `.id` property on plugins which matched the underscore replacement of the `.id` property on actions (and similarly a `get_import_path()` method).